contracts/SafeL2.sol

```diff
@@ -36,9 +36,9 @@ contract SafeL2 is Safe {
     event SafeModuleTransaction(address module, address to, uint256 value, bytes data, Enum.Operation operation);
 
     /**
-     * @inheritdoc ISafe
+     * @inheritdoc Safe
      */
```

Comment:
With this, can the `ISafe` import be removed?

Reply:
Actually yes.